client/.env.tmpl

```diff
@@ -0,0 +1,7 @@
+# This is your .env file
+# The data added here will be propagated to the client
+# example:
+# PORT=8080
+VUE_APP_MENU_SERVICE_URL=${MENU_SERVICE_URL}

```

Comment:
VUE_APP_MENU_SERVICE_URL=${MENU_SERVICE_URL}/menu

Reply:
or another option would be to add it in the Server.js
https://github.com/momander/cymbal-eats/pull/7/files#diff-66079f30b1fcd737aedd8b7b165726706965b9d400907c4bef4112f372a866e0R2

const url = process.env.VUE_APP_MENU_SERVICE_URL+"/menu";